modules/backup-etcd-hosted-cluster.adoc

```diff
@@ -43,11 +43,11 @@ $ oc exec -it <etcd_pod_name> -n <hosted_cluster_namespace> -- env ETCDCTL_API=3
 $ oc exec -it <etcd_pod_name> -n <hosted_cluster_namespace> -- env ETCDCTL_API=3 /usr/bin/etcdctl -w table snapshot status /var/lib/data/snapshot.db
 ----
 
-. Copy the snapshot data to a location where you can retrieve it later, such as an S3 bucket, as shown in the following example.
+. Copy the snapshot data to a location where you can retrieve it later, such as an S3 bucket. See the following example.
 +
 [NOTE]
 ====
-The following example uses signature version 2. If you are in a region that supports signature version 4, such as the us-east-2 region, use signature version 4. Otherwise, if you use signature version 2 to copy the snapshot to an S3 bucket, the upload fails and signature version 2 is deprecated.
+The following example uses signature version 2. If you are in a region that supports signature version 4, such as the `us-east-2` region, use signature version 4. Otherwise, if you use signature version 2 to copy the snapshot to an S3 bucket, the upload fails and signature version 2 is deprecated.
```

Comment:
Reworded

Reply:
`and signature version 2 is deprecated.` is not needed. because the example uses signature version 2 which means for some regions it's not deprecated.